### PR TITLE
Fixes for 0.14.1

### PR DESCRIPTION
**Pull request by koenderks**
Close https://github.com/jasp-stats/jasp-test-release/issues/1073
Close https://github.com/jasp-stats/jasp-test-release/issues/1072
Close https://github.com/jasp-stats/jasp-test-release/issues/1074
Close https://github.com/jasp-stats/jasp-test-release/issues/1076
Close https://github.com/jasp-stats/jasp-test-release/issues/1077
Close https://github.com/jasp-stats/jasp-test-release/issues/1079
Close https://github.com/jasp-stats/jasp-test-release/issues/1080

### CODE REVIEW

**Comment by koenderks** [comment]
I will make sure the unit tests work

File: R/auditCommonFunctions.R [hunk 1]
```diff
@@ -2019,17 +2019,97 @@
...
+                                  N = N,
+                                  increase = options[["sampleSizeIncrease"]],
+                                  prior = prior)
+          n[i] <- result[["sampleSize"]]
+          k[i] <- result[["expectedSampleError"]]
+          progressbarTick()
+        }
+      })
+      
+      if(class(leftPlotError) != "try-error"){
```
Comment (Kucharssim) [inline R/auditCommonFunctions.R:2062]:
```suggestion
      if(!isTryError(leftPlotError)){
```
(function from `jaspBase`)

File: R/auditCommonFunctions.R [hunk 2]
```diff
@@ -2040,104 +2120,47 @@
...
-        result <- jfa::planning(materiality = performanceMateriality,
-                                confidence = confidence,
-                                likelihood = parentOptions[["likelihood"]],
-                                expectedError = i - 1,
-                                N = N,
-                                minPrecision = minPrecision,
-                                prior = prior,
-                                increase = options[["sampleSizeIncrease"]])
-        n[i] <- result[["sampleSize"]]
+        if(jaspBase:::.extractErrorMessage(rightPlotError) == "Sample size could not be calculated, you may want to increase the maxSize argument."){
```
Comment (Kucharssim) [inline R/auditCommonFunctions.R:2158]: should this be `gettext`ed?

File: R/auditCommonFunctions.R [hunk 3]
```diff
@@ -3474,79 +3497,88 @@
...
+        x.labels <- format(jaspGraphs::getPrettyAxisBreaks(seq(min(values), 1.1 * max(values), length.out = 100), min.n = 4), scientific = FALSE)
+        yLimits <- c(min(values), 1.1 * max(values))
+      }
+      
+    })
     
-    if(options[["display"]] == "displayValues"){
-      x.labels <- format(jaspGraphs::getPrettyAxisBreaks(seq(0, 1.1 * max(values), length.out = 100), min.n = 4), scientific = FALSE)
-      values.labels <- paste(prevOptions[["valuta"]], ceiling(values))
+    if(class(plotError) != "try-error"){
```
Comment (Kucharssim) [inline R/auditCommonFunctions.R:3561]: again `!isTryError()` perhaps...

**Review by Kucharssim — APPROVED** [review]
Looks great, the bugs seem to be fixed, I just left three minor comments.

File: R/auditCommonFunctions.R (hunk 2 above)
Comment (koenderks) [reply R/auditCommonFunctions.R:2158]: I don't think so, this is an error message returned by the `jfa` package.

**Pull request merged**